<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #151f2e;
color: #FFFFFF;
font: 11pt &quot;Nirmala UI&quot;;


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="login_label">
       <property name="text">
        <string>Login:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="login_edit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #515f75;
	border-radius: 6px;
    padding: 5px;
	background-color: #252e3d;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_edit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #515f75;
	border-radius: 6px;
    padding: 5px;
	background-color: #252e3d;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirm_button">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #252e3d; 
    border-radius: 6px;
	padding-top: 5px;
	padding-bottom: 5px;
}
</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
